Ginger/GingerCore/Drivers/Appium/SeleniumAppiumDriver.cs

```diff
@@ -927,37 +992,37 @@ private void MobileDeviceActionHandler(ActMobileDevice act)
 
                     case ActMobileDevice.eMobileDeviceAction.PressBackButton:
                         PressBackBtn();
-                        DriverWindow.ShowActionEfect(true, 200);
+                        if (DriverWindow != null) DriverWindow.ShowActionEfect(true, 200);
                         break;
 
                     case ActMobileDevice.eMobileDeviceAction.PressHomeButton:
                         PressHomebtn();
-                        DriverWindow.ShowActionEfect(true, 200);
+                        if (DriverWindow != null) DriverWindow.ShowActionEfect(true, 200);
                         break;
 
                     case ActMobileDevice.eMobileDeviceAction.PressMenuButton:
                         PressMenubtn();
-                        DriverWindow.ShowActionEfect(true, 200);
+                        if (DriverWindow != null) DriverWindow.ShowActionEfect(true, 200);
                         break;
 
                     case ActMobileDevice.eMobileDeviceAction.SwipeDown:
                         SwipeScreen(eSwipeSide.Down);
-                        DriverWindow.ShowActionEfect(true, 300);
+                        if (DriverWindow != null) DriverWindow.ShowActionEfect(true, 300);
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6113827)